### DISCUSSION TITLE
Can you collect Sysmon data from computer B and send it to computer A?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10497] [Unsupported Versions]
Hi, I have two laptops. I have security onion installed on laptop A. I can see Sysmon data from that laptop in Kibana.  I am wondering if I could install Sysmon and winlogbeat on laptop B and send that data to laptop A to see that data in Kibana?

thx

**Comment by dougburks** [comment]
Yes. From https://docs.securityonion.net/en/2.3/introduction.html (emphasis added):
_In the diagram below, we see Security Onion in a traditional enterprise network with a firewall, workstations, and servers. You can use Security Onion to monitor north/south traffic to detect an adversary entering an environment, establishing command-and-control (C2), or perhaps data exfiltration. You’ll probably also want to monitor east/west traffic to detect lateral movement. As more and more of our network traffic becomes encrypted, it’s important to fill in those blind spots with additional visibility in the form of endpoint telemetry. **Security Onion can consume logs from your servers and workstations so that you can then hunt across all of your network and host logs at the same time**._
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/6ada943e-6d33-42e2-8628-ee71be1e5dc6)

**Reply by iqworks** [reply]
thanks doug, will give this a try and will let you know my progress.
